NETOBSERV-906 convert and post-process flows spec (#279)
netobserv/network-observability-operator: docs/flows-format.adoc

// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
:_content-type: REFERENCE
[id="network-observability-flows-format-specifications_{context}"]
= Network Flows format specifications

Description::
+

> This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export.

Reviewer: ```suggestion
This is the specification of the network flows format, used both internally and when exporting flows to Kafka.
```

Reviewer: Also one question, when you refer to using internally, are you referring to using to query Loki?

+

> The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki, as _Labels_, unlike _Fields_, must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].

Reviewer: ```suggestion
The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki, as _Labels_, unlike _Fields_, which must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
```

Author: I'm not sure of that one? In its second part, the sentence is already with subject=Labels and verb=must so I don't think "which" would be correct here? (It could be reformulated as: "This distinction only matters when querying Loki. This is because _Labels_, unlike _Fields_, must be used in etc.". )

Reviewer: Okay, I think your reworking, "This distinction only matters when querying Loki. This is because Labels, unlike Fields, must be used in etc." is good, lets go with that.

+

> Outside of Loki context, for instance when reading this specification as a documentation for the Kafka export feature, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them at the same level in the JSON structure.

Reviewer: ```suggestion
When using this specification as a reference for exporting flows to Kafka, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them at the same level in the JSON structure.
```

== Labels

> === SrcK8S_Namespace

Reviewer: I'm getting asciibinder errors with these H3's. I was wondering about making these list items, as you can see here: https://github.com/openshift/openshift-docs/pull/56335/files

wdyt?

Author: Sure
Do you see anything else that should be changed? The format is dictated by `typedoc`, so I can't do absolutely anything I want without too much pain, but some little things can certainly be adapted. For instance, I was wondering if the line separators `'''` were really useful or if we should remove them.. wdyt?

Author: When looking in my editor renderer, the "+" seem important to make all the description appear at the same level of indentation

Reviewer: Ah, I see. Is the 'Description::' a necessary element for your build? I think we can also do without the 'Description::' and leave the text as introductory text that follows the H1 "Network Flows format specifications". 

Another thing to mention about the Description is that I want to relocate this sentence: "This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export" so that it is introductory text for the Assembly file. Like this: 
![image](https://user-images.githubusercontent.com/77501157/222264328-5bc3599b-7805-476f-8781-66554fb427f3.png)

Author: As discussed, I have changed the generated file to make it an ASSEMBLY file
